test/libsolidity/semanticTests/ecrecover/ecrecover.sol

```diff
@@ -7,8 +7,8 @@ contract test {
 // compileViaYul: also
 // ----
 // a(bytes32,uint8,bytes32,bytes32):
-// 0x18c547e4f7b0f325ad1e56f57e26c745b09a3e503d86e00e5255ff7f715d3d1c,
-// 28,
-// 0x73b1693892219d736caba55bdb67216e485557ea6b6af75f37096c9aa6a5a75f,
+// 0x18c547e4f7b0f325ad1e56f57e26c745b09a3e503d86e00e5255ff7f715d3d1c, 
+// 28, 
+// 0x73b1693892219d736caba55bdb67216e485557ea6b6af75f37096c9aa6a5a75f, 
```

Comment:
I think we should make isoltest not insert trailing whitespace here.

Reply:
Not sure why it does it.